docs/source/hpc.rst

```diff
@@ -44,6 +44,39 @@ The following system software requirements are the minimum for any new or existi
 |              |             |   WCOSS2                              |
 +--------------+-------------+---------------------------------------+
 
+===========================
+Feature availability by HPC
+===========================
```

Comment:
So this feels like something that shouldn't be in the HPC section and should instead be in either the "GFS Configuration" or perhaps "Global Workflow Components" section or become a new section. We should separate the feature/science stuff from the purely HPC stuff.

Also, I see the new "Wave Component" section...that should move under the "Global Workflow Components" section. Perhaps we can move it in this PR as well or a follow-up one?

Thoughts? @WalterKolczynski-NOAA @aerorahul

Reply:
I think the Wave Component section should be moved as well.  There's a lot going on in that section and some of it could be expanded to other components.

I would suggest making a component subsection for each of the major forecast/DA components (ATM, Waves, Ocean/Ice, Aerosols) with submodule names (e.g. Ocean DA via the GDASApp and forecasting via the UFS/MOM6), valid grids, and associated fix files.  This could probably be a table.  Then, maybe under the "Contributing to the Global Workflow" section, we could add subsections to cover adding new grids and fix files.

The GFS Configuration section is in need of a complete revamp, but it would be difficult to maintain the graphics as JEDI-based DA jobs are rapidly changing.  I wonder if there is a way to code a flowchart in either RST or a Python-drawn image/set of images.  That would make it much easier to update.

I still think the capabilities matrix fits well in the HPC Settings section, but it could refer back to the Global Workflow Components subsections.

Aside from the GFS Configuration section, I think everything could be tackled in this PR.  I can open an issue for the flowcharts if there is consensus on going that way.

Reply:
I think a lot of the docs need to be rethought for the new multiply coupled world. Fix stuff here if it is easy, but otherwise let's not bog this PR down with correcting a bunch of ancillary things.

Reply:
Yeah, I'm finding I have opened a can of worms.  I'll make a couple of cleanup changes here and open another issue to rework the documentation.

Reply:
Works for me!

Reply:
I think I am as happy as I'm going to be with this table now.  I don't like how long the code had to be, but it was either this, wonky headers, or raw HTML.  I'm OK with returning to wonky headers if this is too much.